bittorrent/params.go

```diff
@@ -54,6 +54,22 @@ type QueryParams struct {
 	infoHashes []InfoHash
 }
 
+type routeParamsKey struct{}
+
+// RouteParamsKey is a key for the context of a request that
+// contains the named parameters from the http router
```

Comment:
Godoc comments should be full sentences, including a period at the end.

Reply:
Thanks for the catch. Latest push fixes this.